ios/KMMIC/Sources/Presentation/Modules/SurveyDetail/Models/SurveyDetailUIModel.swift

```diff
@@ -14,18 +14,21 @@ struct SurveyDetailUIModel: Equatable {
     let title: String
     let description: String
     let coverImageURL: URL?
+    let detail: SurveyDetail?
```

Comment:
Don't need to have this data here

Reply:
@suho updated at [279d78c](https://github.com/markgravity/kmm-ic/pull/88/commits/279d78ce7c4f986afed3b97d8f4e79fab139fd9c)